src/action/base/create_or_merge_nix_config.rs

```diff
@@ -409,13 +409,10 @@ impl Action for CreateOrMergeNixConfig {
             new_config.push('\n');
         }
 
-        new_config.push_str(
-            &[
-                "# Generated by https://github.com/DeterminateSystems/nix-installer.",
-                "# See `/nix/nix-installer --version` for the version details.",
-            ]
-            .join("\n"),
-        );
+        new_config
+            .push_str("# Generated by https://github.com/DeterminateSystems/nix-installer.\n");
+        new_config.push_str("# See `/nix/nix-installer --version` for the version details.\n");
+        new_config.push_str("\n");
```

Comment:
I don't think this final newline is necessary. Configs will look like

```
# Generated by https://github.com/DeterminateSystems/nix-installer.
# See `/nix/nix-installer --version` for the version details.

config-here
```

rather than 

```
# Generated by https://github.com/DeterminateSystems/nix-installer.
# See `/nix/nix-installer --version` for the version details.
config-here
```

It doesn't really matter at the end of the day, but I myself prefer the second over the first.

Reply:
I like it aesthetically so lets just roll with it hah